.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "lib/forge-std"]
-	path = lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
```

Comment:
instead of maintaining two versions of forge-std, figured we can just use v4-core's forge-std. its a bit more to update with some of the cheatcodes